python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/load-balance.yml

```
@@ -0,0 +1,89 @@
+                  keys:
+                    udp:
+                      type: dict
+                      description: |-
+                        UDP-specific fields used in the load balancing hash.
+                        Requires EOS version 4.33.1F or higher.
+                      keys:
+                        dst_port:
+                          type: int
+                          required: true
+                          min: 0
+                          max: 65535
+                          description: Use the UDP destination port as a hash input.
+                          convert_types:
+                            - str
+                        payload_bytes:
+                          type: str
+                          description: |-
+                            Specifies the UDP payload bytes to use in hash calculation.
+                            Accepts single bytes (e.g., "10"), comma-separated bytes (e.g., "0,1,5"),
+                            ranges (e.g., "0-15"), or combinations (e.g., "0-10,12,15,20-25").
+                            Valid values are between 0 and 62.
+                          convert_types:
+                            - int
+                        match:
+                          type: dict
+                          description: Configuration to match specific bits and define custom payload-based hashing.
+                          keys:
+                            payload_bits:
+                              type: str
+                              required: true
+                              description: |-
+                                Specifies the bit positions within the UDP payload to match for hashing.
+                                Accepts a single bit (e.g., "12"), a comma-separated list (e.g., "0,3,8"),
+                                a range (e.g., "0-15"), or combinations (e.g., "0-7,9,12-15").
+                                Valid values must be in the range 0 to 503.
+                                Matching is limited to a maximum of 16 bits total.
+                              convert_types:
+                                - int
+                            pattern:
+                              type: str
+                              required: true
+                              description: |-
+                                Bit pattern to match in the UDP payload. The value must match the number of bits defined in `payload_bits`.
+                                The valid range is from 0 to (2^N - 1), where N is the number of bits selected in `payload_bits`.
+                                Matching is supported only on up to 16 bits of the UDP payload.
+                                Supports both integer and hexadecimal values for this field.
+                                Integer inputs will be internally converted to their hexadecimal representation by EOS.
+                              convert_types:
+                                - int
```

@MaheshGSLAB following a discussion with Claus

```suggestion
                              type: str
                              required: true
                              description: |-
                                Bit pattern to match in the UDP payload. 
                                The value should be given as an hexadecimal format `0x...`
                                The valid range is from 0 to (2^N - 1), where N is the number of bits selected in `payload_bits`.
```

I don't think this `Matching is supported only on up to 16 bits of the UDP payload.` applies for this field right?

---

updated